.github/workflows/deploy.yml

```diff
@@ -0,0 +1,152 @@
+name: deploy
```

Comment:
The deploy script probably can be removed, datta is a library-only crate for now.

Reply:
Good catch!